app/src/main/res/layout/fragment_movie.xml

```diff
@@ -24,20 +22,62 @@
         android:id="@+id/view_search_button"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Button"
+        android:text="@string/button_text"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/view_search" />
 
-    <TextView
-        android:id="@+id/view_content_video"
+     <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/layout_content"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="16dp"
+        android:layout_height="350dp"
```

Comment:
Używając dużych rozmiarów w dp trzeba pamiętać, że widok będzie miał trochę różny rozmiar na różnych telefonach (zależy to głównie od rozmiaru i rozdzielczości ekranu). Jeśli nasza aplikacja nie wspiera widoku horyzontalnego, bezpieczniejszą opcją jest użycie w takim przypadku rozmiarów procentowych. Tu jest artykuł o tym: https://wajahatkarim.com/2018/04/percentage-width/height-using-constraint-layout/